LEAF_Request_Portal/admin/ajaxIndex.php

```diff
@@ -281,6 +315,11 @@ function checkToken()
 
         $resHistory = $type->getHistory($itemID);
 
+        for($i = 0; $i<count($resHistory); $i++){
+            $dateInLocal = new DateTime($resHistory[$i]['timestamp'], new DateTimeZone('UTC'));
+            $resHistory[$i]["timestamp"] = $dateInLocal->setTimezone(new DateTimeZone($tz))->format('Y-m-d H:i:s T');;
```

Comment:
There's an extra ";" at the end of the line.

Reply:
Done